sdk/src/main/java/io/dapr/actors/client/ActorProxyHttpAsyncClient.java

```diff
@@ -4,24 +4,33 @@
  */
 package io.dapr.actors.client;
 
-import io.dapr.client.AbstractDaprHttpClient;
+import io.dapr.client.ClientRequestBuilder;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
 import io.dapr.utils.Constants;
 import okhttp3.OkHttpClient;
 import reactor.core.publisher.Mono;
 
 /**
  * Http client to call actors methods.
  */
-class ActorProxyHttpAsyncClient extends AbstractDaprHttpClient implements ActorProxyAsyncClient {
+class ActorProxyHttpAsyncClient implements ActorProxyAsyncClient {
+
+    private DaprClient daprClient;
 
     /**
      * Creates a new instance of {@link ActorProxyHttpAsyncClient}.
      *
      * @param port       Port for calling Dapr. (e.g. 3500)
      * @param httpClient RestClient used for all API calls in this new instance.
      */
-    ActorProxyHttpAsyncClient(int port, OkHttpClient httpClient) {
```

Comment:
The idea is to share the same httpClient for all instances. With this change, each instance will have its own client, which is expensive.

Reply:
so for sharing the same httpClient I should be gettting the acual implementation rather than the builder?